### ISSUE TITLE
Near zero upload speeds with PIA Wireguard in Unraid

### ISSUE THREAD

**Issue by penguincrest** [binhex/arch-qbittorrentvpn#285]
I've searched as much as I could to try to find a resolution for this before posting.  After downloading healthy torrents I'm getting near zero upload speeds and it appears as if peers just aren't connecting.  I'm using the Toronto PIA endpoint in the list of supported port forwarder endpoints for PIA Wireguard.  Looking at the startup logs it seems to be detecting the port and updating it correctly in qbittorrent.   I can pull my public IP from the VPN connection the container is making and go to yougetsignal.com and see that the port assigned for my IP is actually open.  What am I missing and what logs can I pull to help investigate.  I appreciate the help!

**Comment by bradarussell** [comment]
I’ve had issue trying to use Toronto.  Try ca-montreal and see if it’s better.  I don’t know if it was my config at the time or the Toronto server but I never got it working well.  Ontario seems intermittent for seeding as well.  Montreal has seemed the best for me.  You may also want to try some of the geo located servers like Bahamas.

**Comment by CheekyPanda2** [comment]
yeah try switching to Montreal. I've been using Montreal and it seems mostly fine.
